tools/tf2ngraph.py

```diff
@@ -343,7 +421,16 @@ def main():
     args = prepare_argparser(allowed_formats)
     inp_format, inp_loc = filter_dict("input", args.__dict__)
     out_format, out_loc = filter_dict("output", args.__dict__)
-    output_nodes = args.output_nodes.split(',')
+    if args.output_nodes is None:
+        possible_out_nodes = infer_output_nodes(inp_format, inp_loc)
+        print(
+            "Analysed graph for possible list of output nodes. Please supply one or more output node in --output_nodes"
+        )
+        for out_node in possible_out_nodes:
+            print(out_node, "of type", possible_out_nodes[out_node])
```

Comment:
Print this with quotes. Currently the output looks like:
```
Analysed graph for possible list of output nodes. Please supply one or more output node in --output_nodes
num_detections of type Identity
detection_boxes of type Identity
detection_scores of type Identity
detection_classes of type Identity
Traceback (most recent call last):
...
```
Instead, consider changing this to:
```
Name: `num_detections` Type: `Identity`
Name: `detection_boxes` Type: `Identity`
...
```
Give a space between the final line and the beginning of the stack trace.